### ISSUE TITLE
WebIDL section links no longer work?

### ISSUE THREAD

**Issue by rtoy** [tabatkins/bikeshed#1593]
The WebAudio spec has entries like `[[WebIDL#idl-Uint8Array|Uint8Array]]`.  These have worked fine in the past but today, they give errors (see https://travis-ci.com/WebAudio/web-audio-api/builds/144943905 and WebAudio/web-audio-api#2125):

>+LINE ~4228: Couldn't find section '#idl-Uint8Array' in spec 'webidl':
>+[[WebIDL#idl-Uint8Array|Uint8Array]]
>+LINE ~4273: Couldn't find section '#idl-Uint8Array' in spec 'webidl':
>+[[WebIDL#idl-Uint8Array|Uint8Array]]
>+LINE ~4306: Couldn't find section '#idl-Float32Array' in spec 'webidl':
>+[[WebIDL#idl-Float32Array|Float32Array]]
>+LINE ~4334: Couldn't find section '#idl-Float32Array' in spec 'webidl':
>+[[WebIDL#idl-Float32Array|Float32Array]]

Can't tell if these links were always incorrect or if bikeshed is broken.

**Comment by tabatkins** [comment]
Huh, I'll have to see why WebAudio was missed when I went thru all my tests and filled issues on all the specs that would break due to this change.

Anyway, yes, the links were always wrong - they're section autolinks (meant to point to a heading), but being used to point to definitions. If Bikeshed doesn't know about the heading data in a spec, it just assumes you know what you're doing and makes a link with ugly default text.

But it turns out I was just handling the spec names wrong, so Bikeshed was assuming it knew nothing about some specs when it actually totally did. I just fixed that, so now it's (properly) complaining that those IDs don't match any heading. 😀

For example, all of those are IDL links, and should be `{{UInt8Array}}`, etc.
